content/docs/command-reference/import-url.md

```diff
@@ -1,8 +1,9 @@
 # import-url
 
 Download a file or directory from a supported URL (for example `s3://`,
-`ssh://`, and other protocols) into the <abbr>workspace</abbr>, and track
-changes in the remote data source. Creates a `.dvc` file.
+`ssh://`, and other protocols) into either the <abbr>workspace</abbr> or to a
```

Comment:
we still create .dvc file in this case, right? I would not put this special way of importing into this synopsis part

Reply:
could you please briefly explain why do we need to support `--to-remote` in both - `dvc add` and `dvc import-url`? Just want to be sure I understand this. Why `dvc add` is not enough?

Reply:
I have the same Q. Maybe iterative/dvc/issues/4527 is a better route for imports?

Reply:
> could you please briefly explain why do we need to support --to-remote in both - dvc add and dvc import-url?

Initially it was only in the `import-url`, though for users who don't want to track the original location we went ahead and also applied the same thing for the `add`.

Reply:
Hmmm. I thought `add` was the original/main command here, per iterative/dvc/issues/4520. Anyway, I def. see how it makes sense as an "alternative" to `add --external` and thinking about it, iterative/dvc/issues/4527 is not enough (to transfer to remote directly) in the case of `import-url`.

What about plain `import` though? Shouldn't it be included as well? Basically to move specific data from one remote to another, while at the same time tracking it independently in a new project (a workaround to [chained imports](https://github.com/iterative/dvc/issues/3305)).